rocAL/rocAL/source/rali_api_augmentation.cpp

```diff
@@ -1647,7 +1647,7 @@ RALI_API_CALL raliCropMirrorNormalize(RaliContext p_context, RaliImage p_input,
         context->capture_error(e.what());
         ERR(e.what())
     }
-    return input; // Changed to input----------------IMPORTANT
+    return output;
```

Comment:
why is this reverted back?

Reply:
This was a bug Rajy which was introduced long time back.

This did not affect any of our trainings because the Crop Mirror Normalizes output is never passed to any other node as an input

Hence reverted back the change